File: src/main/java/me/viniciusarnhold/altaria/apis/imdb/Imdb.java
package me.viniciusarnhold.altaria.apis.imdb;

import com.omertron.imdbapi.ImdbApi;
import org.jetbrains.annotations.NotNull;


/**
 * Created by Vinicius.
 *
 * @since ${PROJECT_VERSION}
 */
public class Imdb {


    @NotNull
    private static ImdbApi ourInstance = new ImdbApi();

    private Imdb() {
    }


    @NotNull
    public static ImdbApi api() {
        return ourInstance;
    }
}

File: core/src/main/kotlin/me/viniciusarnhold/altaria/apis/imdb/Imdb.kt
package me.viniciusarnhold.altaria.apis.imdb

import com.omertron.imdbapi.ImdbApi


/**
 * Created by Vinicius.

 * @since 1.0
 */
object Imdb {


    private val ourInstance = ImdbApi()


    fun api(): ImdbApi {
        return ourInstance
    }
}
